.editorconfig

```diff
@@ -8,5 +8,8 @@ insert_final_newline=true
 indent_style=space
 indent_size=4
 
+[*.yml]
+indent_size=2
```

Comment:
Can we keep 4? It's really unsettling to have a vertical spacing as wide as the indentation :(

Reply:
Personally, I would always use 2 spaces for indentation with YAML files as 4 spaces makes it rather awkward when working with definitions of objects in lists.

I can change to 4 spaces, though, if that is a requirement.

Note that we currently have mixed indentation - sometimes 2, sometimes 4 spaces.

Reply:
Guess so... Could we uniformize it somehow? might be more proper to do it in another PR - whatever the choice is

Reply:
Two spaces work best for YML because lists call for two. 
```
list:
  - foo: 10
    bar: 20
  - foo: 11
    bar: 21
```

With four spaces you'll need to manually pad them:

```
list:
    - foo: 10
      bar: 20
    - foo: 11
      bar: 21
```